sender.go

```diff
@@ -323,6 +323,32 @@ func (s *LineSender) Int64Column(name string, val int64) *LineSender {
 	return s
 }
 
+// Long256Column adds a 256-bit unsigned integer (long256) column
+// value to the ILP message.
+// 
+// val is pure unsigned hexadecimal string. Any attempt to set a string
```

Comment:
Could you change the first sentence to something like `val should contain a hex encoded 256-bit unsigned integer value with "0x" prefix and "i" suffix.`?